apps/judicial-system/backend/src/app/modules/case/case.controller.ts

```diff
@@ -297,12 +298,6 @@ export class CaseController {
         break
       case CaseTransition.SUBMIT:
         if (isIndictmentCase(theCase.type)) {
-          if (!user.canConfirmIndictment) {
```

Comment:
Moved this logic to the `prosecutorTransitionRule` as only prosecutors are supposed to be able to submit cases and permission checks should be in roles rules and guards when possible.